$NetBSD: patch-Documentation_Makefile,v 1.2 2020/03/19 22:30:30 rillig Exp $

* Use BSD_INSTALL_*.
* FIXME: Add "--skip-validation" to xmlto command.

--- Documentation/Makefile.orig	2014-11-26 22:39:03.000000000 +0000
+++ Documentation/Makefile
@@ -107,7 +107,7 @@ ASCIIDOC_COMMON = $(ASCIIDOC) $(ASCIIDOC
 TXT_TO_HTML = $(ASCIIDOC_COMMON) -b $(ASCIIDOC_HTML)
 TXT_TO_XML = $(ASCIIDOC_COMMON) -b $(ASCIIDOC_DOCBOOK)
 MANPAGE_XSL = manpage-normal.xsl
-XMLTO = xmlto
+XMLTO = xmlto --skip-validation
 XMLTO_EXTRA =
 INSTALL ?= install
 RM ?= rm -f
@@ -228,16 +228,16 @@ pdf: user-manual.pdf
 install: install-man
 
 install-man: man
-	$(INSTALL) -d -m 755 $(DESTDIR)$(man1dir)
-	$(INSTALL) -d -m 755 $(DESTDIR)$(man5dir)
-	$(INSTALL) -d -m 755 $(DESTDIR)$(man7dir)
-	$(INSTALL) -m 644 $(DOC_MAN1) $(DESTDIR)$(man1dir)
-	$(INSTALL) -m 644 $(DOC_MAN5) $(DESTDIR)$(man5dir)
-	$(INSTALL) -m 644 $(DOC_MAN7) $(DESTDIR)$(man7dir)
+	$(BSD_INSTALL_MAN_DIR) $(DESTDIR)$(man1dir)
+	$(BSD_INSTALL_MAN_DIR) $(DESTDIR)$(man5dir)
+	$(BSD_INSTALL_MAN_DIR) $(DESTDIR)$(man7dir)
+	$(BSD_INSTALL_MAN) $(DOC_MAN1) $(DESTDIR)$(man1dir)
+	$(BSD_INSTALL_MAN) $(DOC_MAN5) $(DESTDIR)$(man5dir)
+	$(BSD_INSTALL_MAN) $(DOC_MAN7) $(DESTDIR)$(man7dir)
 
 install-info: info
-	$(INSTALL) -d -m 755 $(DESTDIR)$(infodir)
-	$(INSTALL) -m 644 git.info gitman.info $(DESTDIR)$(infodir)
+	$(BSD_INSTALL_DATA_DIR) $(DESTDIR)$(infodir)
+	$(BSD_INSTALL_DATA) git.info gitman.info $(DESTDIR)$(infodir)
 	if test -r $(DESTDIR)$(infodir)/dir; then \
 	  $(INSTALL_INFO) --info-dir=$(DESTDIR)$(infodir) git.info ;\
 	  $(INSTALL_INFO) --info-dir=$(DESTDIR)$(infodir) gitman.info ;\
